analyzers/src/SonarAnalyzer.Common/Rules/Utilities/SymbolReferenceAnalyzerBase.cs

```diff
@@ -42,22 +41,21 @@ public abstract class SymbolReferenceAnalyzerBase<TSyntaxKind> : UtilityAnalyzer
 
         protected sealed override SymbolReferenceInfo CreateMessage(SyntaxTree syntaxTree, SemanticModel semanticModel)
```

Comment:
`SyntaxTree syntaxTree` should be renamed into `tree` and `SemanticModel semanticModel` should be renamed into `model`. Also, I think to remember that the semantic model parameter should come as first, although I am not sure this rule applies to all cases.

Reply:
Indeed. I took a note and I will create a separate PR for this since the names come from the base class `UtilityAnalyzerBase` and I will have to change all the implementing classes.

Reply:
Done in https://github.com/SonarSource/sonar-dotnet/pull/7965